backend/Pipfile

```diff
@@ -12,6 +12,7 @@ django-cors-headers = "==3.13.0"
 django-filter = "==22.1"
 uvicorn = {version = "==0.18.3", extras = ["standard"]}
 gunicorn = "==20.1.0"
+djangorestframework-simplejwt = "*"
```

Comment:
Can you pin this dependency to the specific version?

Reply:
> 

good catch. Pinning to 5.2.0